react-ai-chatbot/src/components/SettingsButton/SettingsButton.jsx

```
@@ -0,0 +1,23 @@
+import styles from "./SettingsButton.module.css";
+
+export function SettingsButton({ onClick }) {
+  return (
+    <button className={styles.Button} onClick={onClick}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)